src/ansi-c/expr2c_class.h

```diff
@@ -208,6 +228,9 @@ class expr2ct
   std::string convert_designated_initializer(const exprt &src, unsigned &precedence);
   std::string convert_concatenation(const exprt &src, unsigned &precedence);
   std::string convert_sizeof(const exprt &src, unsigned &precedence);
+
+  std::string convert_struct(const exprt &src, unsigned &precedence,
```

Comment:
So I can correctly reapply this, what is the rule here? Is it because one of the parameters is on a new line therefore they all should be?